persistence/upgrades/ravendb-5to6.md

```diff
@@ -165,6 +165,59 @@ In NServiceBus versions 3 and 4, RavenDB Persistence was part of the main NServi
 
 If attempting to convert directly from NServiceBus 4 or lower to NServiceBus 7 (which is _not recommended_) timeout data will have to be converted to the new format manually.
 
+### Patching legacy timeouts
+
+The following patch can be used to idempotently update all timeout documents to the new format.
+
+```javascript
+from TimeoutDatas  
+update {
+    if(!this.Destination.Queue) {
+        return;
+    }
+    if(!this.Destination.Machine) {
+        this.Destination = this.Destination.Queue
+    }
+    else {
+        this.Destination = this.Destination.Queue + "@" + this.Destination.Machine
+    }
+}
+```
+
+If only timeout data using the new format is present nothing will be changed. If old timeout documents are present they will be converted into the new format.
```

Comment:
I would avoid conditional ("if") style language here. The point is that this is safe to run. Customers don't need to worry about whether this applies to them or not.

```suggestion
Only timeout documents with with the old format are modified. Timeout documents already in the new format are not modified.
```